en/admin/config.rst

```diff
@@ -566,6 +570,63 @@ The following are parameters related to the database server. The type and value
   
     **tcp_keepalive** is a parameter which specifies if you apply SO_KEEPALIVE option to TCP network protocol or not. The default is **yes**. If this value is **no**, DB server-side connection can be disconnected when transaction logs are not copied for a long time in the firewall environment between master and slave.
  
+
+**use_user_hosts** 
+
+    **use_user_hosts** is a system paramter to use to select the host look up between **hostname** and **IP address**, required by the CUBRID service from the services below. The default value is **OFF**.
+
+    * The host/IP address look-up library that **OS** provides. (**glibc**, Linux)
+    * **The host/IP address look-up library** that **CUBRID** provides.
+
+    use_user_hosts=off (default)
+
+    * Looks up between IP address and hostname using by system library.
+    * In general, provides /etc/hosts host look-up, DNS Query commonly
+
+    use_user_hosts=on
+
+    * Looks up between IP address and hostname using by CUBRID host look-up library.
```

Comment:
I think it is better to say "Looks up between IP address and hostname using CUBRID host look-up library."

Reply:
I will edit manual what you suggest. Thank you!